### DISCUSSION TITLE
Can't access web-interface of my Security Onion

### DISCUSSION THREAD

**Discussion by mayank-joshi-01** [Security-Onion-Solutions/securityonion#11645] [Unsupported Versions]
I successfully installed the standalone version of Security onion but I can access it from my host machine, I don't know why

![so-status](https://github.com/Security-Onion-Solutions/securityonion/assets/85271575/e34004a3-8d3a-4627-aa78-e1f036f04b37)

this is my so-status everything works fine I guess.

![so-allow](https://github.com/Security-Onion-Solutions/securityonion/assets/85271575/3f620035-db36-4ba4-b0b8-7018db5ffa89)

I am using Chrome and it shows me 

"This site can’t be reached"

I also tried pinging the virtual machine(Security Onion) from the IPI get to access the web Interface

Thanks in advance.

**Comment by cm-ops** [comment]
Is the IP you are using allowed in the analyst hostgroup? `salt-call pillar.get firewall:hostgroups:analyst`

**Reply by mayank-joshi-01** [reply]
Don't know about Hostgroup sir, but I added a group and my home netword IP using so-firewall

**Reply by cm-ops** [reply]
Are you able to ping the IP from your host machine? If you ran the command I put above, do you see your IP in the output? What did you select as your url_base? `salt-call pillar.get global:url_base`

**Reply by mayank-joshi-01** [reply]
Sorry for late reply, but I can't ping my virtual machine and my base url it https://10.0.2.15

**Reply by cm-ops** [reply]
Is that IP in the analyst firewall config in SOC?

**Reply by dougburks** [reply]
@Alex01000100 From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

**Reply by Alex01000100** [reply]
Sorry. Done.
